src/main/java/jenkins/plugins/hipchat/HipChatNotifier.java

```diff
@@ -280,7 +280,7 @@ private HipChatService getHipChatService(AbstractBuild<?, ?> build) {
                 desc.getSendAs());
     }
 
-    private static HipChatService getHipChatService(String server, String token, boolean v2Enabled, String room,
+    public static HipChatService getHipChatService(String server, String token, boolean v2Enabled, String room,
```

Comment:
It's a goof practice to have javadocs for public methods. Maybe it's not very important in this case